CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 #### Fixes
 
+* [#2412](https://github.com/ruby-grape/grape/pull/2412): Update outdated documentation for XML formatter - [@eriklovmo](https://github.com/eriklovmo)
```

Comment:
This is missing a period. But no need for a CHANGELOG entry for a small doc change, feel free to remove it.

Reply:
Ah, better to leave it out then, just adds clutter. Will do.